<launch>
    <arg name="language" default="fr"/>  <!-- Options: fr or en -->
    <arg name="single_task_per_person" default="true"/>

    <node pkg="smart_speaker" type="smart_speaker_smart_node" name="smart_speaker_smart_node" output="screen" launch-prefix="bash -c 'sleep 30; $0 $@' ">
        <param name="language" value="$(arg language)"/>

        <param name="person_distance_threshold" value="1.5"/>
        <param name="person_distance_frame_id" value="odas"/>
        <param name="nose_confidence_threshold" value="0.4"/>

        <param name="video_analysis_message_count_threshold" value="5"/>
        <param name="video_analysis_message_count_tolerance" value="2"/>

        <param name="use_after_task_delay_duration_topic" value="true"/>
        <param name="after_task_delay_duration_s" value="30"/> <!-- Useless when use_after_task_delay_duration_topic=true -->
        <param name="single_task_per_person" value="$(arg single_task_per_person)"/>

        <rosparam param="songs">
            - name: Ça fait rire les oiseaux
              keywords:
                - rire
                - oiseau
              path: ~/.ros/ttop/Compagnie_créole_ca_fait_rire_les_oiseaux_30s.mp3
            - name: La dame en bleu,
              keywords:
                - dame
                - bleu
              path: ~/.ros/ttop/Michel_Louvain_La_Dame_En_Bleu_30s.mp3
            - name: Un amour qui ne veut pas mourir,
              keywords:
                - amour
                - mourir
              path: ~/.ros/ttop/Renee_Martel_Un_Amour_Qui_Ne_Veut_Pas_Mourir_30s.mp3
        </rosparam>
    </node>

    <include file="$(find t_top)/launch/bringup.launch">
        <arg name="language" value="$(arg language)"/>
        <arg name="slam" value="false"/>
        <arg name="fullscreen_face" value="true"/>
    </include>
</launch>
